components/header/header.hbs

```diff
@@ -1,8 +1,25 @@
-<div class="container">
-  <header class="header">
-      <div class="logo header__logo">
-          <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg" alt="Office for National Statistics" class="logo__img">
-      </div>
-  </header>
-</div>
-{{>@bar bar_title="eQ Title bar"}}
+<header class="header {{classes}}">
+    <div class="header__top" role="banner">
+        <div class="container">
+            <div class="grid grid--gutterless">
+                <div class="grid__col col-5@m">
+                    <div class="logo">
+                        <a href="#1"><img src="{{path '/assets/img/ons-logo-white.svg'}}" alt="Office for National Statistics logo" class="logo__img header__logo"></a>
```

Comment:
The link has a hit area that is smaller than that of the logo .svg.

- Firefox `18.5px` high vs `24px` high
- Chrome `21px` high vs `23.98px` high
- Safari `21px` high vs `24px` high

Note: With CSS disabled the logo becomes illegible as the background colour is removed.